base/sparse/linalg.jl

```diff
@@ -532,10 +531,13 @@ function normestinv{T}(A::SparseMatrixCSC{T}, t::Integer = 2)
     end
```

Comment:
Sorry, I got the wrong line. I meant:
```javascript
     if t > n
         throw(ArgumentError("number of blocks must not be greater than $n"))
     end
```
It seems to me that we are guaranteed that `t` can't be greater than `n` at this point.